orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -191,6 +196,11 @@ def _get_session_config(self):
                 provider_option_map["cudnn_conv_algo_search"] = self._runtime_options.conv_algo_search
                 provider_option_map["cudnn_conv_use_max_workspace"] = "1"
                 provider_option_map["cudnn_conv1d_pad_to_nc1d"] = "1"
+                if self._runtime_options.enable_tuning:
+                    provider_option_map["tunable_op_enable"] = "1"
+                    provider_option_map["tunable_op_tuning_enable"] = "1"
+                elif self._runtime_options.tuning_results_path:
+                    provider_option_map["tunable_op_enable"] = "1"
```

Comment:
why not add "tunable_op_tuning_enable" too?

Also, shall we set the max tuning duration as well? Default can be 0 (which is the current default as well)

Reply:
tunable_op_enable is to enabling the tunning future, including offline and online. tunable_op_tuning_enable is to enable online tunning. The logic in elif is for offline tunning.

Reply:
@askhade The workflow is as follows:

### Online
- tunable_op_enable=1: use op index in tuning context
- tunable_op_tuning_enable=1: allow to run FindFastest

Then TunableOp -> FindFastest -> Save results to tuning context -> dispatch to the fastest one, on every TunableOp dispatch.

And sess creators call `get_tuning_results` at their disposal to take a snapshot for tuning context.

### Offline
- tunable_op_enable=1
- tunable_op_tuning_enable=0:  will not run FindFastest

TunableOp -> dispatch to the one in tuning context, on every TunableOp dispatch.

And sess creators call `set_tuning_results` at their disposal to restore a snapshot for tuning context.

### TunableOp disabled
- tunable_op_enable=0

TunableOp -> dispatch to the one hardcoded with default id, usually 0, on every TunableOp dispatch.

The default one should be the generic one.